oracle/README.md

```diff
@@ -254,14 +227,10 @@ When running the processes, the following commands can be used to test functiona
 
 | Variable | Description |
 |-------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `HOME_RPC_URL` | The HTTPS URL(s) used to communicate to the RPC nodes in the Home network. |
-| `FOREIGN_RPC_URL` | The HTTPS URL(s) used to communicate to the RPC nodes in the Foreign network. |
 | `USER_ADDRESS` | An account - the current owner of coins/tokens. |
```

Comment:
These parameters are for scripts that are using for testing. So, a separate `.env` file could be used there - cut copy of the `.env` file used to run the bridge. That's why it is important either to list the corresponding parameters here one more time or to have its own set of parameters that will be used only for testing.

Reply:
Done - I have rolled back the removal of these lines, only updating with proper prefix.